sdk/python/setup.py

```diff
@@ -19,7 +19,7 @@
     "pytest-tornasync",
     "mypy",
     "black==24.3.0",
-    "flake==4.0.1",
+    "flake8==4.0.1",
```

Comment:
What is this change?
I guess that you failed to rebase your branch 🧐

Reply:
Yeah I'm having a lot of problems right now because of signing, sorry. Give me a little bit to fix this